.travis.yml

```diff
@@ -1,4 +1,6 @@
 language: scala
 jdk:
 - oraclejdk8
-script: sbt +test
+script: 
+  - sbt +test
```

Comment:
Done! Note that the standard Travis build command is `sbt ++$TRAVIS_SCALA_VERSION test`. Maybe we should that instead?